src/federation-info-plugin/components/SchemaFieldsTable/SchemaFieldsTable.jsx

```diff
@@ -0,0 +1,121 @@
+import React, { useState } from 'react'
+import { TableRow, TableCell, TableSortLabel } from '@mui/material'
+
+import SchemaOperationTable from '../SchemaOperationTable/SchemaOperationTable'
+
+import {
+  fieldArgsToValue,
+  fieldTypeToValue,
+  fieldOwnerServicesToValue,
+  fieldReferencedByToValue
+} from '../../utils/schemaFieldToTableCellValue'
+
+const FieldHeader = ({ showReference, order, orderBy, createSortHandler }) => {
+  return (
```

Comment:
memoize all the: `createSortHandler ('field')` here I think would be the most efficent.

Reply:
Commit pushed based on the solution we discussed on the call.